redash/docker-compose.yaml

```
@@ -0,0 +1,61 @@
+    image: redash/redash:10.1.0.b50633
+    command: server
+    depends_on:
+      - redash-postgres
+      - redash-redis
+    ports:
+      - "5500:5000"
+    env_file: .env
+    environment:
+      REDASH_WEB_WORKERS: 4
+      PYTHONUNBUFFERED: 0
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)